File: wwwroot/js/venue.js
ab.beerQuest.venue = function (venue) {
    var self = this;

    self.id = ko.observable(venue.id);
    self.name = ko.observable(venue.name);
    self.category = ko.observable(venue.category);
    self.url = ko.observable(venue.url);
    self.date = ko.observable(venue.date);
    self.excerpt = ko.observable(venue.excerpt);
    self.thumbnail = ko.observable(venue.thumbnail);
    self.lat = ko.observable(venue.lat);
    self.lng = ko.observable(venue.lng);
    self.address = ko.observable(venue.address);
    self.phone = ko.observable(venue.phone);
    self.twitter = ko.observable(venue.twitter);
    self.starsBeer = ko.observable(venue.starsBeer);
    self.starsAtmosphere = ko.observable(venue.starsAtmosphere);
    self.starsAmenities = ko.observable(venue.starsAmenities);
    self.starsValue = ko.observable(venue.starsValue);
    self.tags = ko.observableArray(venue.tags);
    
}

File: DataModel/Venue.cs
using AB.BeerQuest.DataModel.Enums;

namespace AB.BeerQuest.DataModel
{
    public record Venue
    {
        public Venue()
        {
            Tags = new List<VenueTag>();
        }

        public int Id { get; set; }
        public string? Name { get; set; }
        public Category Category { get; set; }
        public string? Url { get; set; }
        public DateTime Date { get; set; }
        public string? Excerpt { get; set; }
        public string? Thumbnail { get; set; }
        public decimal Lat { get; set; }
        public decimal Lng { get; set; }
        public string? Address { get; set; }
        public string? Phone { get; set; }
        public string? Twitter { get; set; }
        public float StarsBeer { get; set; }
        public float StarsAtmosphere { get; set; }
        public float StarsAmenities { get; set; }
        public float StarsValue { get; set; }
        public ICollection<VenueTag> Tags { get; set; }
    }
}
